joomla/joomla-cms
# com_cache (cache view) implement searchtools and ordering

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the search tools (filter, order, etc) to com_cache cache view.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/13034157/cdbbaadc-d325-11e5-8645-a42466df0825.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/13034179/7071e3a4-d326-11e5-9de9-69421bd09e1d.png)
#### How to test
1. Install latest staging
2. Go to System -> Global configuration and enable some type of caching
3. Naviagt in the backend and in the frontend
4. Go to System -> Clear Cache, you will see some cached items in "site" and in "administrator"
5. Apply this PR
6. Go to System -> Clear Cache
7. Test the new searchtools (search, filter, client selector, ordering, pagination)
